native/libcst/src/nodes/traits.rs

```diff
@@ -170,28 +170,13 @@ pub mod py {
                 .map(|x| x.try_into_py(py))
                 .collect::<PyResult<Vec<_>>>()?
                 .into_iter();
-            Ok(PyTuple::new_bound(py, converted).into())
-        }
-    }
-
-    impl TryIntoPy<PyObject> for PyTuple {
```

Comment:
I found that with IntoPyObject that some of these impls aren't needed at all. I didn't look too closely to try to understand why though so there could be something subtle happening.

Reply:
Great, I was hoping to get rid of these.